vmms/ec2SSH.py

```diff
@@ -380,3 +380,9 @@ def existsVM(self, vm):
             if inst.instances[0].id is vm.ec2_id:
                 return True
         return False
+
+    def getImages(self):
+        """ getImages - Lists all images in TASHI_IMAGE_PATH that have the
+        .img extension
+        """
+        return ["test.img"]
```

Comment:
Maybe just return an empty list? Is there a particular reason a test.img is returned?